src/release_artifacts_resources/ios/cdk/cdk/distribution/s3_construct.py

```diff
@@ -0,0 +1,15 @@
+from aws_cdk import aws_s3, core
+
+
+class S3Construct(core.Construct):
+    def __init__(self, scope: core.Construct, construct_id: str, **kwargs) -> None:
+        super().__init__(scope, construct_id, **kwargs)
+
+        bucket_name = "aws-sdk-ios-release-prod"
+
+        self.bucket = aws_s3.Bucket(
```

Comment:
Whoops, we need to set the retain policy for the bucket as well! Include a similar warning comment as above with CloudFront. I can't remember whether S3 does that with a separate `apply_removal_policy` or as an argument to the initializer.

Reply:
S3 doesnot destroy the bucket by default - https://docs.aws.amazon.com/cdk/api/latest/docs/aws-s3-readme.html#bucket-deletion . Do I need to add it explicitly?

Reply:
Added `RETAIN`